src/pages/home/ReportScreen.js

```diff
@@ -113,7 +113,7 @@ const defaultProps = {
  * @returns {String}
  */
 function getReportID(route) {
-    return route.params.reportID.toString();
+    return String(lodashGet(route, 'params.reportID', null));
```

Comment:
Should the default value be null here? shouldnt it be ''? If null then we need to update the returns docs too, right?

Reply:
Oh I completely missed that its still wrapped in string :D thanks